tools/isobuild/meteor-npm.js

```diff
@@ -777,7 +777,7 @@ Profile("meteorNpm.runNpmCommand", function (args, cwd) {
           });
         }
       );
-    });
+    }).await();
```

Comment:
{@benjamn,@tmeasday} I am certain about seeing the benefit in that not a single build has timed out with the `lintJshint`/`jshint` issue since that commit has landed in `devel`.  (I also ran it 50+ times on my own CI).

I do not currently understand why the additional `.await()` is necessary either.  Digging around for the exact reason was too much of a time sink (i.e. sporadicity of the reproduction) and I didn't see a danger with doubling up on it until it could be given more attention.

Perhaps the entire `execFile` code here should be replaced with the (more thorough/robust) `execFile*` methods from [`/tools/utils/processes.js`](https://github.com/meteor/meteor/blob/9304f76b52b02ee9c484737c7571a7d39042369d/tools/utils/processes.js#L56)

I probably should have added a code comment about it though, or perhaps leave an issue for it.  I'll try to take a look at it again sometime, but currently the deeper `Fiber`+`Promise` stuff is out of my league.